go/test/endtoend/cluster/cluster_util.go

```diff
@@ -265,7 +265,7 @@ func WaitForReplicationPos(t *testing.T, tabletA *Vttablet, tabletB *Vttablet, v
 		msg := fmt.Sprintf("%s's replication position to catch up to %s's;currently at: %s, waiting to catch up to: %s", tabletB.Alias, tabletA.Alias, replicationPosB, replicationPosA)
 		select {
 		case <-ctx.Done():
-			assert.FailNowf(t, "Timeout waiting for condition '%s'", msg)
+			assert.FailNow(t, "Timeout waiting for condition '%s'", msg)
```

Comment:
this change doesn't look right

Reply:
@systay It actually is! It looks confusing, but `Fallnowf` expects first a message and then a separate interpolation thingy. Just `Failnow` without the `f` already interpolates for the first string it's passed. It's confusing.

Reply:
Going to pull this out as neither works as expected here and don't want to churn on this in this PR.